lib/MusicBrainz/Server/Report/EditorReport.pm

```diff
@@ -16,7 +16,7 @@ around inflate_rows => sub {
     return [
         map +{
             %$_,
-            editor => $editors->{ $_->{id} }
+            editor => $editors->{ $_->{id} }->TO_JSON,
```

Comment:
This is not new from this patch, but can you fix MBS-11423 while you're changing this? It needs unsanitized data (emails).